src/main/java/teammates/common/util/Const.java

```diff
@@ -24,7 +24,6 @@
     public static final String USER_NOBODY_TEXT = "-";
     public static final String USER_UNKNOWN_TEXT = "Unknown user";
     public static final String TEAM_OF_EMAIL_OWNER = "'s Team";
-    public static final String REGEXP_TEAM = String.format("^.*%s$", TEAM_OF_EMAIL_OWNER);
```

Comment:
Added in 35ee389... handled team responses (1 year, 3 months ago) to determine whether a ResponseAttribute's giver is a team (e.g., if response.giverEmail is `"alice.b@tmmt.com's team"`).

No longer used now because now the team name is stored in the giverEmail if the giver type is team